### PR TITLE
Add instructions for building on Mac OS X.

### PR DESCRIPTION
**Pull request by judah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,36 @@ There is also a demo application:
...
+
+- Build the TensorFlow library and install it on your machine:
+
+        cd third_party/tensorflow
+        ./configure  # Choose the defaults when prompted
+        bazel build -c opt tensorflow:libtensorflow_c.so
+        cp bazel-bin/tensorflow/libtensorflow_c.so /usr/local/lib
+        cd ../..
+
+- Append the following lines to this project's `stack.yaml`:
```
Comment (blackgnezdo) [inline README.md]: We should just do this. Our docker image relies on it too.

File: README.md [hunk 2]
```diff
@@ -22,3 +22,36 @@ There is also a demo application:
...
+
+- Install dependencies via [Homebrew](http://brew.sh):
+
+        brew install swig
+        brew install bazel
+
+- Build the TensorFlow library and install it on your machine:
+
+        cd third_party/tensorflow
+        ./configure  # Choose the defaults when prompted
```
Comment (blackgnezdo) [inline README.md]: third_party/tensorflow/tensorflow/tools/docker/Dockerfile.devel
does this
tensorflow/tools/ci_build/builds/configured CPU

I wonder which one is right?

Comment (judah) [inline README.md]: That script just calls ./configure, and its comments suggest it's a hack specific to Docker.  So I think `./configure` is more appropriate here.  (It's also what the other language bindings recommend.)

File: README.md (hunk 1 above)
Comment (judah) [inline README.md]: Done.

**Pull request merged**